src/converter/avro/src/commonMain/kotlin/community/flock/wirespec/converter/avro/AvroEmitter.kt

```diff
@@ -47,13 +47,9 @@ object AvroEmitter {
                 } else {
                     def.also { hasEmitted.add(def.identifier.value) }.emit(ast, hasEmitted)
                 }
-
                 is Enum -> def.emit()
-                is Channel -> TODO()
-                is Endpoint -> TODO()
-                is Refined -> TODO()
-                is Union -> TODO()
-                null -> AvroModel.SimpleType(value)
+                is Refined -> AvroModel.SimpleType("string")
+                else -> AvroModel.SimpleType(value)
```

Comment:
Let's also ensure we emit unions. I doubt this works as is here.

Reply:
Unions now also Emit